Content.Server/_Goobstation/StationEvents/GameDirectorSystem.cs

```diff
@@ -0,0 +1,576 @@
+
+                var ruleEnt = GameTicker.AddGameRule(randomAntag.ID);
+                GameTicker.StartGameRule(ruleEnt);
+            }
+            else
+            {
+                if (calmStartAntags.Count < 2)
+                {
+                    LogMessage("Not enough valid roundstart antags found!");
+                    return;
+                }
+
+                LogMessage("Choosing multiple roundstart antags");
+                var random = new Random();
+
+                var firstIndex = random.Next(calmStartAntags.Count);
+                var randomAntagFirst = calmStartAntags[firstIndex];
+                calmStartAntags.RemoveAt(firstIndex);
+
+                var randomAntagSecond = calmStartAntags[random.Next(calmStartAntags.Count)];
+
+                LogMessage($"Roundstart antags chosen: 1: {randomAntagFirst.ID} 2: {randomAntagSecond.ID}");
+
+                var ruleFirstEnt = GameTicker.AddGameRule(randomAntagFirst.ID);
+                var ruleSecondEnt = GameTicker.AddGameRule(randomAntagSecond.ID);
+
+                GameTicker.StartGameRule(ruleFirstEnt);
+                GameTicker.StartGameRule(ruleSecondEnt);
+            }
+    }
+
+    /// <summary>
+    ///   Count the active players and ghosts on the server.
+    ///   Players gates which stories and events are available
+    ///   Ghosts can be used to gate certain events (which require ghosts to occur)
+    /// </summary>
+    public PlayerCount CountActivePlayers() // DeltaV - Allow admins to force the story to change
+    {
+        var allPlayers = _playerManager.Sessions.ToList();
+        var count = new PlayerCount();
+        foreach (var player in allPlayers)
+        {
+            // TODO: A
+            if (player.AttachedEntity != null)
+            {
+                // TODO: Consider a custom component here instead of HumanoidAppearanceComponent to represent
+                //        "significant enough to count as a whole player"
+                if (HasComp<HumanoidAppearanceComponent>(player.AttachedEntity))
+                    count.Players += 1;
+                else if (HasComp<GhostComponent>(player.AttachedEntity))
+                    count.Ghosts += 1;
+            }
+        }
+
+        return count;
+    }
+
+    /// <summary>
+    ///   Count all the players on the server.
+    /// </summary>
+    public int GetTotalPlayerCount(IList<ICommonSession> pool)
+    {
+        var count = 0;
+        foreach (var session in pool)
+        {
+            if (session.Status is SessionStatus.Disconnected or SessionStatus.Zombie)
+                continue;
+
+            count++;
+        }
+
+        return count;
+    }
+
+    /// <summary>
+    ///   Sorts the possible events and then picks semi-randomly.
+    ///   when maxRandom is 1 it's always the "best" event picked. Higher values allow more events to be randomly selected.
+    /// </summary>
+    protected RankedEvent SelectBest(List<RankedEvent> bestEvents, int maxRandom)
```

Comment:
'GameDirectorSystem.SelectBest(List<RankedEvent>, int)': new protected member declared in sealed type
```suggestion
    private RankedEvent SelectBest(List<RankedEvent> bestEvents, int maxRandom)
```